topcoffea/modules/datacard_tools.py

```diff
@@ -31,15 +31,22 @@ def to_hist(arr,name,zero_wgts=False):
     # NOTE:
     #   If we don't instantiate a new np.array here, then clipped will store a reference to the
     #   sub-array arr and when we modify clipped, it will propagate back to arr as well!
-    clipped = np.array(arr[1:-1])     # Strip off the under/overflow bins
-    clipped[-1] += arr[-1]  # Add the overflow bin to the right most bin content
-    nbins = len(clipped)
+    clipped=[]
+    for i in range(2): # first entry is sum(weight), second entry is sum(weight^2)
+        if arr[i] is not None:
+            clipped.append( np.array(arr[i][1:-1]))     # Strip off the under/overflow bins
+            clipped[i][-1] += arr[i][-1]  # Add the overflow bin to the right most bin content
+        else: 
+            clipped[i]=None
```

Comment:
In the same vein as some of my later comments, I feel like this block of code can be made to be more `numpy`-like, but the array manipulations here are sufficiently complex that I would need to sit down and make sure to get the syntax right. As it is (mostly) just an aesthetic/styling change, I'm fine with skipping over it for now and revising it later.